book/logging.md

```diff
@@ -11,85 +29,148 @@ regulatory requirements and uncovering the root causes of incidents.\
 
+For more information, see:
+* [Open Shift Documentation - Cluster Logging](
+  https://docs.openshift.com/container-platform/4.13/logging/cluster-logging.html)
+* [Open Shift Cluster Logging Operator Github repository](
+  https://github.com/openshift/cluster-logging-operator)
+
+
+### AWS VPC Flow Logs
+
+VPC Flow Logs is a feature provided by Amazon Web Services (AWS) that enables to capture
+information about the IP traffic going to and from network interfaces in VPCs.
+
+VPC Flow Logs include:
+
+1. **Traffic to and from the Internet:** This includes traffic that goes through Internet
+Gateways, NAT Gateways, or Virtual Private Gateways (if using VPN or Direct Connect).
+
+2. **Traffic Between Subnets:** VPC Flow Logs also capture traffic between different
+subnets within the same VPC. This includes both inbound and outbound traffic.
+
+3. **Traffic Between Instances:** Traffic between instances, whether they are in the same
+subnet or different subnets, is captured by VPC Flow Logs. This helps monitor
+communication between instances for security and troubleshooting purposes.
+
+4. **Security Group and Network ACL Rules:** Flow Logs indicate whether network traffic
+was accepted or rejected based on the rules of security groups and network ACLs.
+
+5. **Metadata:** Flow Logs capture metadata about the traffic, including source and
+destination IP addresses, source and destination ports, protocol, packet and byte counts,
+etc.
 
-## Splunk
+6. **Communication with AWS Services:** If your instances communicate with other AWS
+services that are accessible over the internet, the traffic to and from those services
+is also captured.
+
+For more information see the AWS docs about 
+[Logging IP traffic using VPC Flow Logs](
+    https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html).
+
+In the scope of RHTAP, VPC Flow Logs are being forwarded to Splunk. \
+The Terraform configuration and documentation reside in the following repository:\
+[Terraform-aws-splunk](https://github.com/eisraeli/terraform-aws-splunk)
+
+
+## Monitoring and Analytics Platforms
+
+Platforms such as Splunk and Amazon CloudWatch, are designed to collect, store, and
+analyze various types of data, including logs, metrics, events, and other forms of data
+generated by different systems. Their primary goal is to provide insights into system
+performance, operational efficiency, application behavior, and other operational aspects. 
+
+### Splunk
 [Splunk](https://splunk.com) is a widely used log management and analytics platform that
 allows organizations to collect, index, search, analyze, and visualize machine-generated
 data, including logs from various sources. Splunk provides powerful search capabilities,
 real-time monitoring, alerting, and robust data visualization options.
 
-For RHTAP, we use a Splunk Cloud, which is a cloud-based version of the Splunk platform.
+For RHTAP, we use Splunk Cloud, which is a cloud-based version of the Splunk platform.
```

Comment:
Are you planning to generalize this for AppStudio?

Reply:
not sure about this, do you expect to merge this in a state that is not suitable for inclusion in the AppStudio book?